Legion/EyeOfAzshara/Trash.lua

```diff
@@ -48,8 +53,10 @@ function mod:GetOptions()
 		195284, -- Undertow
 
 		--[[ Hatecoil Arcanist & Ritualist Lesha ]]--
-		196027 -- Aqua Spout
+		196027, -- Aqua Spout
+		{197105, "HEALER"} -- Polymorph: Fish

```

Comment:
Please add the trailing comma here as well.
There is no reason not to add it, and it improves the readability of diffs.
Should've been there before.